cardano-node/cardano-node.cabal

```diff
@@ -14,6 +14,14 @@ extra-source-files:    README.md, ChangeLog.md
 Flag unexpected_thunks
   Description:   Turn on unexpected thunks checks
   Default:       False
+  
+Flag with_wallet-client
+  Description:   Turn on wallet-client build
+  Default:       True
+
+Flag with_chairman
+  Description:   Turn on chairman build
+  Default:       True
```

Comment:
These should both be set to `Manual: True` too, so that they're not subject to automatic selection by the solver. Otherwise they could be disabled by the solver if one of their deps cannot be satisfies, which would be confusing.

Also, I suggest we stick to `-` in the flag names, rather than mixing `_` and `-`.

Reply:
Hi,
- `Manual: True` has been added to both flags
- `with_wallet-client` has been renamed to `with_wallet_client`, I initially named it `with_wallet-client` to reflect the exact name of the `executable` target defined, but if *underscored* naming convention is the preferred way, it's fine and I'll keep it in mind for the future

I also rebased the commits so it's going to be only one commit even without squashed merging. Also PR description has been updated to reflect the modifications.